api/helpers/game-states/validate-gamestate.js

```diff
@@ -20,16 +22,12 @@ function validateTurn(param) {
 }
 
 function validatePhase(param) {
-  if (
-    param === null ||
-    param === undefined ||
-    Number.isNaN(param) ||
-    param < 0 ||
-    (param > 5 && param !== 7)
-  ) {
-    throw new Error('The phase must be a number in [1, 2, 3, 4, 5, 7]');
+  const phases = Object.values(GamePhase);
```

Comment:
We were previously hard-coding the allowed game phases here, but they're already defined on the `GamePhase` object. It's better to validate them by direct comparison to that object as the source of truth

Reply:
Way cleaner!